packages/cozy-harvest-lib/src/components/AccountModal.jsx

```diff
@@ -140,7 +140,7 @@ export class AccountModal extends Component {
               trigger={trigger}
               account={account}
               onAccountDeleted={onDismiss}
-              addAccount={() => history.push('../new')}
+              addAccount={() => history.push('../../new')}
```

Comment:
It seems a bit brittle to use relative routes like that, do you think it would be possible/better to use an absolute path ?

Reply:
A test for this would help prevent errors.

Reply:
> It seems a bit brittle to use relative routes like that, do you think it would be possible/better to use an absolute path ?

I honestly don't think it's more fragile than using absolute paths. In this case we changed the structure of the routes and forgot to change a link… regardless of absolute or not this would have been broken.

While it's possible to use absolute routes, I'm convinced it's in fact not better. Harvest is like procedures, it has its own routes that are mounted on a "root" route porovided by the application. In procedures we used absolute routes, which means that every bit of code that does a redirect needs to be aware of the "root url" provided by the app. With relatives routes, components only need to be aware of their location within harvest, which I think makes more sense and find easier to maintain so far.

Reply:
> it has its own routes that are mounted on a "root" route porovided by the application. In procedures we used absolute routes, which means that every bit of code that does a redirect needs to be aware of the "root url" provided by the app. 

OK interesting, I had not thought about the "root URL" problem. This is true that components should not know about this. An alternative: in the context of harvest, it might make more sense for components to be passed a scoped `history`, or a `history` that would be aware of the "mount point" of harvest.

```
history.push('<harvest-root>/new')
```

This is not in the context of the PR but I am interested in your thoughts about this.

Reply:
I… quite like that idea? As a dev we'd need to be aware not to use `withRouter`, but other than that it might be a better solution than relative routes. They *do* work, but the DX isn't as smooth as I expected, even with react-router v4. Your solution doesn't even require an upgrade to v4. I'll keep it in mind 👍